storage/blockdevice/COMPONENT_SPIF/include/SPIF/SPIFBlockDevice.h

```diff
@@ -221,10 +221,12 @@ class SPIFBlockDevice : public mbed::BlockDevice {
     /* SFDP Detection and Parsing Functions */
     /****************************************/
     // Send SFDP Read command to Driver
-    int _spi_send_read_sfdp_command(mbed::bd_addr_t addr, void *rx_buffer, mbed::bd_size_t rx_length);
+    int _spi_send_read_sfdp_command(mbed::bd_addr_t addr, mbed::sfdp_cmd_addr_size_t addr_size,
```

Comment:
Shouldn't this change be done in the commit "SFDP: Add more parameters to the reader callback"? Otherwise, I don't think "SFDP: Add more parameters to the reader callback" will even build.

Reply:
Squashed them